### DISCUSSION TITLE
create multi route tour in next js using react-joyride

### DISCUSSION THREAD

**Discussion by ZeeshanMustfai** [gilbarbara/react-joyride#1000] [General]
I have a next js web app. And app has 5 pages with client-side rendering and I want to implement a tour using react-joyride into the whole app. 

Currently, I have implemented it one by one page, and it's working fine. But I need to implement a tour that will start from the first page to the last page smoothly.

**Comment by gilbarbara** [comment]
Hey @ZeeshanMustfai 
This isn't a bug with the library but an implementation question.
There's an example of multi-route in the demo. Check it out.

**Reply by ZeeshanMustfai** [reply]
Hey @gilbarbara 
I have checked the multi-route demo and implemented it in my app successfully. 
Thanks.
